tier2_standards/git_workflow.md

```diff
@@ -18,6 +18,8 @@ While we do not require teams to use any specific workflow, we highly recommend
 
 We require that the chosen `git` workflow is fully documented somewhere within the purview of the repository (e.g. the repository's `README`, a GitHub wiki page, a `CONTRIBUTING.md` file, etc.).  If teams opt to use a workflow that is already documented elsewhere (e.g. the 'branching' or 'forking' workflow in STScI Style Guides), they may simply add a small statement somewhere within their project documentation that indicates which workflow is being used, with appropriate links to the existing documentation.  If teams opt to use a custom workflow, we require that the project documentation contain a full description of the workflow, with step-by-step instructions on how to apply it (similar to the Style Guides workflows).
 
+If teams find that the 'branching' or 'forking' workflow in the STScI Style Guides are close to what they want to use, but not exactly what they want to use, they are welcome to propose changes to the Style Guides document by either contacting Matthew and Shannon with their ideas, or submitting a pull request in the Style Guides repository.
```

Comment:
Maybe it'd say "...but not exactly what they want to use (e.g. teams would like line by line commands), they are welcome..." so they don't go to the style guide team with procedure changes. Also, I cam not a part of the style guide team, so I'm assuming you know that they'd welcome that kind of input from random people at ST? I don't know how their team works so I want to make sure we are sending people they don't want.

Reply:
I think `style_guides` is just considered an open-source project and anyone is welcome to propose changes through pull requests.  I believe Arfon Smith and Steve Crawford used to oversee that effort, but since they are gone/leaving now, I'm not sure what will happen 🤷  But thank you, I put in your suggestion!